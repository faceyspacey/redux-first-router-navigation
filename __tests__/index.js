import navigation from '../src'

test('first', () => {
  const config = navigation({
    stateKey: { im: 'a navigator' }
  })

  expect(config).toMatchSnapshot()
})
