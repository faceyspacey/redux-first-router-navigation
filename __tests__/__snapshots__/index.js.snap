// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`first 1`] = `
Object {
  "actionToNavigation": [Function],
  "navigationToAction": [Function],
  "navigators": Object {
    "stateKey": Object {
      "im": "a navigator",
    },
  },
  "patchNavigators": [Function],
}
`;
